src/command/docs/shortlinks.rs

```diff
@@ -46,20 +137,54 @@ mod tests {
 
     #[test]
     fn can_get_url_from_slug() {
-        let expected_link = "https://go.apollo.dev/r/start";
+        let expected_link = "https://go/apollo/dev/r/start";
```

Comment:
lol idk if i fixed this one on my branch
```suggestion
        let expected_link = "https://go.apollo.dev/r/start";
```

Reply:
Okay, fixed this on my branch.

Reply:
No worries -- I will investigate!